contractsSol5/mock/MockNetwork.sol

```diff
@@ -39,6 +39,10 @@ contract MockNetwork is KyberNetwork {
         (_networkFeeBps, _expiryBlock) = readNetworkFeeData();
     }
 
+    function mockGetNetworkFee() public view returns(uint networkFeeBps) {
```

Comment:
instead of mock function
can use this API

getAndUpdateNetworkFee

Reply:
need to test a line in this function mock. if we use getAndUpdateNetworkFee that line could not be reached